projects/storefrontapp-e2e-cypress/cypress/integration/accessibility/tabbing-order.e2e-spec.ts

```diff
@@ -41,20 +42,22 @@ context("Tabbing order - tests don't require user to be logged in", () => {
     cy.window().then(win => win.sessionStorage.clear());
   });
 
+  describe('Home page', () => {
+    it('should allow to navigate with tab key', () => {
+      homeTabbingOrder(config.home);
+    });
+  });
+
   describe('Footer', () => {
     it('should allow to navigate with tab key', () => {
       footerTabbingOrder(config.footer);
     });
   });
 
   describe('Login page', () => {
-    it('should allow to navigate with tab key (empty form)', () => {
+    it('should allow to navigate with tab key', () => {
       loginTabbingOrder(config.login);
     });
-
-    it('should allow to navigate with tab key (filled out form)', () => {
```

Comment:
I wanted you to delete boolean from the functions parameters and not the whole tests :P